optaplanner-examples/src/main/java/org/optaplanner/examples/investment/solver/move/factory/InvestmentQuantityTransferMoveIteratorFactory.java

```diff
@@ -88,7 +90,7 @@ public boolean hasNext() {
         }
 
         @Override
-        public Move next() {
+        public Move<InvestmentSolution> next() {
```

Comment:
I think we can have this return InvestmentQuantityTransferMove instead of Move<InvestmentSolution>